apps/web/src/pages/CommunityInterests/sections/AdditionalInformation.tsx

```diff
@@ -0,0 +1,92 @@
+import { useIntl } from "react-intl";
+import ClipboardDocumentCheckIcon from "@heroicons/react/24/outline/ClipboardDocumentCheckIcon";
+
+import { Heading } from "@gc-digital-talent/ui";
+import { RichTextInput } from "@gc-digital-talent/forms";
+import { getLocale } from "@gc-digital-talent/i18n";
+
+import { FRENCH_WORDS_PER_ENGLISH_WORD } from "~/constants/talentSearchConstants";
+
+const TEXT_AREA_MAX_WORDS_EN = 100;
+const TEXT_AREA_MAX_WORDS_FR = Math.round(
+  TEXT_AREA_MAX_WORDS_EN * FRENCH_WORDS_PER_ENGLISH_WORD,
+);
+
+export interface SubformValues {
+  additionalInformation: string | null;
+}
+
+interface AdditionalInformationProps {
+  formDisabled: boolean;
+}
+
+const AdditionalInformation = ({
+  // RichTextInput can't be disabled: #12531
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  formDisabled,
+}: AdditionalInformationProps) => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+
+  const wordLimits = {
+    en: TEXT_AREA_MAX_WORDS_EN,
+    fr: TEXT_AREA_MAX_WORDS_FR,
+  } as const;
+
+  return (
+    <div
+      data-h2-display="base(flex)"
+      data-h2-flex-direction="base(column)"
+      data-h2-gap="base(x1.25)"
+    >
```

Comment:
I think this is creating some uneven spacing around the separator. I am not 100% sure if this is the exact div so I included a screenshot to try and illustrate :sweat_smile: 

![2025-02-06_08-18](https://github.com/user-attachments/assets/24d4e088-78a5-4590-9282-2f7af8621445)

Reply:
Oh, OK, I see this.  What's happening is that there are no development programs for the program you selected.  I think we didn't seed any before and I updated the seeder in this PR.  (See testing instructions step 1.)

@JoshBeveridge Should I add a null state well with a message like "There are no development programs associated with this community."?

Reply:
OK, added hiding for that section if there are no development programs
[hide training section if no programs](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12668/commits/172dc8a8034070fffdec9bc227c35de60f0786ea)